test/controllers/landing-page.controller.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// For running our service
+const { init } = require('../../app/server.js')
+
+describe('Landing Page controller', () => {
+  let server
+
+  beforeEach(async () => {
+    // Create server before each test
+    server = await init()
+
+    // We silence calls to server.logger.error in the plugin to keep test output as
+    // clean as possible
+    Sinon.stub(server.logger, 'error')
+
+    // We silence sending a notification to our Errbit instance using Airbrake
+    Sinon.stub(server.app.airbrake, 'notify').resolvesThis()
```

Comment:
Since we are using Sinon to stub the logger and airbrake we need an afterEach block for Sinon.restore()
Have a look at `test/controllers/bills.controller.test.js` to see how to do this

```
  afterEach(() => {
    Sinon.restore()
  })
```